src/System.Private.Xml.Linq/tests/events/System.Xml.Linq.Events.Tests.csproj

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <!-- [ActiveIssue(https://github.com/dotnet/corefx/issues/21986)] causes a read access violation when built in ret. -->
+    <ILCBuildType Condition="'$(ArchGroup)' == 'x86'">chk</ILCBuildType>
```

Comment:
I think this is ok since this is better than not working but I'm not sure if we can ship with chk binaries like this cc: @weshaggard